src/test/java/org/gitlab4j/api/TestProtectedBranchesApi.java

```diff
@@ -157,6 +156,6 @@ public void testProtectBranch() throws GitLabApiException {
         List<ProtectedBranch> branches = gitLabApi.getProtectedBranchesApi().getProtectedBranches(project.getId());
         assertNotNull(branches);
         assertTrue(branches.stream()
-                .anyMatch((protectedBranch) -> protectedBranch.getName().equals(TEST_BRANCH_NAME)));
+                .anyMatch((protectedBranch) -> branch.getName().equals(TEST_BRANCH_NAME)));
```

Comment:
Correct me if I am wrong, but I think this should remain unchanged. I wrote this test to protect an unprotected branch, get a list of protected branches, and verify that the unprotected branch is now part of this list. 

By changing the name, the assertion will always return `true` as `branch` is effectively treated as a `final` variable (line 155) and should always be `TEST_BRANCH_NAME` by the time (line 159) executes. If it does not, (line 155) should have failed with an exception thrown.

Reply:
Not sure how that got changed, fixed and pushed.

Reply:
Made another change to this.  After the revert to protectedBranch the ```branch``` variable was not referenced, and I realized that the contents of the returned protected branch were never really validated, by validating the name, it took care of the unused reference.